### PR TITLE
spotifyd: new package at 0.2.20

### PR DESCRIPTION
**Pull request by alaughlin**
## Description of package
Spotifyd allows third party Spotify clients to be able to stream music without relying on the official client

## Checklist

- [x] Latest upstream version.
- [x] I agree to maintain this package (**required**).
- [x] I agree to notify KISS if I can no longer maintain this package (**required**).

### CODE REVIEW

File: community/spotifyd/build [hunk 1]
```diff
@@ -0,0 +1,4 @@
+#!/bin/sh -e
+
+cargo build --release --no-default-features --features alsa_backend
```
Comment (dylanaraps) [inline community/spotifyd/build]:
```sh
 cargo build \
    --release \
    --no-default-features \
    --features alsa_backend
```

File: community/spotifyd/depends [hunk 2]
```diff
@@ -0,0 +1,2 @@
+rust make
```
Comment (dylanaraps) [inline community/spotifyd/depends]:
```sh
alsa-lib make
rust      make
```

File: community/spotifyd/build (hunk 1 above)
Comment (alaughlin) [reply community/spotifyd/build]: fixed

File: community/spotifyd/depends (hunk 2 above)
Comment (alaughlin) [reply community/spotifyd/depends]: fixed

**Comment by dylanaraps** [comment]
Thanks. :+1:

**Pull request merged**